pkg/dispatchcli/cmd/get_api.go

```diff
@@ -85,9 +85,17 @@ func formatAPIOutput(out io.Writer, list bool, apis []v1.API) error {
 		table.Append([]string{
 			*a.Name, *a.Function,
 			strings.Join(a.Protocols, "\n"), strings.Join(a.Methods, "\n"), strings.Join(a.Hosts, "\n"),
-			strings.Join(a.Uris, "\n"), a.Authentication, string(a.Status), fmt.Sprintf("%t", a.Enabled),
+			strings.Join(addHostName(a.Uris), "\n"), a.Authentication, string(a.Status), fmt.Sprintf("%t", a.Enabled),
 		})
 	}
 	table.Render()
 	return nil
 }
+
+func addHostName(uris []string) []string {
+	var newUris []string
+	for _, uri := range uris {
+		newUris = append(newUris, dispatchConfig.Host + ":8081" + uri)
```

Comment:
How do we display the http endpoint versus the https endpoint? They will certainly have different ports. I think this requires a little more thought in exactly how we fulfill this requirement. I would imagine adding a new command to the api command that gets the base url and displaying those for http and https.